h2o-core/src/main/java/hex/grid/HyperParameters.java

```diff
@@ -0,0 +1,79 @@
+package hex.grid;
+
+import water.AutoBuffer;
+import water.H2O;
+import water.Iced;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+/**
+ * Wrapper class to make hyper-parameters serializable with Icer
+ */
+public class HyperParameters extends Iced<HyperParameters> {
```

Comment:
Should we also store version of H2O the parameters are intended to be users with. ? What if someone upgrades H2O and then attempts to load old GS ?

Reply:
this is stored with Icer, that guarantees that only the same version of H2O will be able to load it